sky/packages/sky/lib/sky_tool

```diff
@@ -251,10 +224,171 @@ class StartSky(object):
                     app_path = os.path.join(args.sky_src_path, args.ios_sim_debug_build_path, IOS_APP_NAME)
                 IOSSimulator.fork_install_app(app_path)
 
+        # TODO(iansf): fix this so that we don't have to pass sky_server_root, main_dart, pid, and args.
+        android.setup_servers(sky_server_root, main_dart, pids, args)
+
+
+class StopSky(object):
+    def add_subparser(self, subparsers):
+        stop_parser = subparsers.add_parser('stop',
+            help=('kill all running SkyShell.apk processes'))
+        stop_parser.set_defaults(func=self.run)
+
+    def _run(self, args):
```

Comment:
I think this is diff noise.  I didn't write this, just made very minor modifications to it.  Good ideas, though!